docs/documentation/telemetry.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: documentation
+title: Telemetry
+doc_order: 100
+description: About the use of Telemetry in mozregression
+---
+
+# Telemetry
+
+Unless explicitly disabled, mozregression will send an anonymized telemetry ping (a small JSON-document) to Mozilla's servers on each use (command-line invocation in the case of console mozregression, each bisection or single run in the case of the GUI).
+
+This data is anonymized and not personally identifiable, it contains only:
+
+- A random client id (e.g. `5a26b9f7-c025-4858-abce-63ca73ceebf0`)
```

Comment:
If you link to the glean docs below, then you can also drop this sentence.

Reply:
Done